README.md

```diff
@@ -1,5 +1,72 @@
-# StrathPort: A School Transport and Vehicle Allocation System
+# StrathPort
+------------------------------------------------------------------------------------------------------------
+A School Transport and Vehicle Allocation System for Strathmore University built using the Laravel Framework.
 
-## About Project
+# About StrathPort
+------------------------------------------------------------------------------------------------------------
+This IS Project focuses on creating a web application for school transport scheduling and vehicle allocation for students, club heads and staff members of Strathmore University in the aim of ensuring there is reliable transportation provided by the school and that there is an efficient means of communicating transportation status and information within the school.This project includes modules like Transport Requests, Transport Schedules, Carpool Driver and Admin Module whereby in the Transport Requests Module, a student head or club head can make a transport request that would require the admin to review it depending on certain criteria like the number of people being requested for a vehicle, availability status of a bus at the time one has requested among others by either approving or rejecting. For the Transport Scheduling Module, the admin gets to update the status of different fixed-route buses that help commuting staff and students everyday. For the Carpool Driver Module, incase of rejection of a transport request, one can opt for carpooling as an option whereby carpooling drivers register and get to accept or decline requests made for certain trips. On the Admin module, they get to carry out advanced CRUD operations such as user management, reviewing of transport requests, update school driver and vehicle details, review the different drivers that register as carpool drivers as well as their carpool vehicle details and update transport schedules for the week.Development of this web application system project is so as to solve the current approach taken into the school transport and vehicle allocation process which lacks a centralized system for managing reliability of such a process causing a significant obstacle to the functioning of daily operations and extracurricular activities.
+
+# Setup and Installation
+------------------------------------------------------------------------------------------------------------
+## Installation Requirements
```

Comment:
This needs to be reformatted to show inline or block code using backticks instead of using single quotes

Reply:
how do i do this?